vendor/k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -658,6 +658,10 @@ const (
 	// Status code 406
 	StatusReasonNotAcceptable StatusReason = "NotAcceptable"
 
+	// StatusReasonRequestEntityTooLarge means that the request entity is too large.
+	// Status code 413
+	StatusReasonRequestEntityTooLarge StatusReason = "RequestEntityTooLarge"
```

Comment:
Added manually as it was missing - had been added before #74000, kubernetes/kubernetes@b971b12

Reply:
just ignore my previous comment, this method was added in https://github.com/openshift/origin/pull/22267/commits/f22e9205fb8e64e37e36be5c96b7eb13f3d55aad